resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/Utils.scala

```diff
@@ -27,4 +27,36 @@ object Utils extends Logging {
     val resource = createResource
     try f.apply(resource) finally resource.close()
   }
+
+  def checkAndGetK8sMasterUrl(rawMasterURL: String): String = {
```

Comment:
Is this the same as https://github.com/apache/spark/blob/a2f502cf53b6b00af7cb80b6f38e64cf46367595/core/src/main/scala/org/apache/spark/util/Utils.scala#L2725?

Reply:
Yes, will fix in next commit

Reply:
Now fixed